codejail/safe_exec.py

```diff
@@ -94,11 +94,7 @@ def not_safe_exec(
     globals_dict.update(json_safe(g_dict))
 
 
-# If the developer wants us to be unsafe (ALWAYS_BE_UNSAFE), or if there isn't
-# a configured jail for Python, then we'll be UNSAFE.
-UNSAFE = ALWAYS_BE_UNSAFE or not is_configured("python")
```

Comment:
This was causing some tests to be skipped (and possibly would have caused issues in production) because it required the `'python'` Jail to be configured at import time.  I've updated the tests to catch this error, and updated this code to not implicitly fall back to unsafe execution.